lib/elixir/lib/calendar/iso.ex

```diff
@@ -218,7 +225,10 @@ defmodule Calendar.ISO do
   @doc """
   Converts the given time into a string.
   """
-  def time_to_string(hour, minute, second, microsecond, format \\ :extended)
+  @impl true
```

Comment:
this is subtle:

```
@impl true
def time_to_string(hour, minute, second, microsecond, format \\ :extended)
```

complained that `time_to_string/4` was not marked with `@impl true`